tachyon/math/polynomials/univariate/radix2_evaluation_domain_unittest.cc

```diff
@@ -49,8 +53,12 @@ TYPED_TEST(Radix2EvaluationDomainTest, CosetLDEBatch) {
     naive.CosetLDEBatch(expected, 1, shift);
     std::unique_ptr<Radix2EvaluationDomain<F>> domain =
         Radix2EvaluationDomain<F>::Create(size_t{1} << log_r);
-    domain->CosetLDEBatch(result, 1, shift);
-    EXPECT_EQ(expected, result);
+    if (log_r == 0) {
+      EXPECT_DEATH(domain->CosetLDEBatch(result, 1, shift), "");
+    } else {
+      domain->CosetLDEBatch(result, 1, shift);
+      EXPECT_EQ(expected, result);
+    }
```

Comment:
This also seems to be changes that don't match the contents of the commit too.

Changing the commit title might also be one way to address this.